test/test_helper.rb

```diff
@@ -0,0 +1,7 @@
+require "minitest"
+require "minitest/autorun"
+require "minitest/reporters"
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
```

Comment:
We don't use minitest/reporters anywhere else in our repo. Can you just omit this (and not depend on the minitest-reporters gem either)